KNOWN_ISSUES.md

```diff
@@ -29,3 +29,10 @@ Unfortunately, there is no easy fix for this, other than re-writing all of the l
 
 Therefore, for the present and near future, Angular Universal will not be supported.
 
+## Using CSS transform: scale makes slider render incorrectly 
+
+Using `transform : scale(N)` in css will mess with the calculation of the slider. You can workaround this by setting the scale in ngx-slider options as the opposite value of the scale set in css. [See Issue.](https://github.com/danisss9/ngx-slider/issues/2)
```

Comment:
This probably needs to be entered and linked as an issue in the main repository.